components/auth-proxy-service/src/main/java/com/wso2telco/proxy/dao/AttShareDAO.java

```diff
@@ -0,0 +1,16 @@
+package com.wso2telco.proxy.dao;
+
+import com.wso2telco.core.dbutils.DBUtilException;
+
+import java.sql.SQLException;
+import java.util.Map;
+
+/**
+ * Created by aushani on 9/20/17.
```

Comment:
Remove ide auto generated comment

Reply:
Removed the auto-generated comments and added WSO2Telco license to the attribute Sharing related classes.